src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx

```diff
@@ -106,61 +107,89 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum }) => {
+        </Button>
+      </OverlayTrigger>
+    );
+  };
+
   return (
     <PageBlock title={'Risk Log'}>
       <Form>
-        {risks.map((risk, idx) => (
-          <div key={idx} className={styles.container}>
-            <Form.Check
-              label={
-                <p
+        <div className={styles.parentContainer}>
+          {risks.map((risk, idx) => (
+            <div key={idx} className={styles.container}>
+              {hasPermissions ? (
+                <Form.Check
+                  label={
+                    <p
+                      style={
+                        risk.isResolved
+                          ? { textDecoration: 'line-through' }
+                          : { textDecoration: 'none' }
+                      }
+                    >
+                      {risk.detail}
+                    </p>
+                  }
+                  checked={risk.isResolved}
+                  data-testId={`testCheckbox${idx}`}
+                  onChange={() => handleCheck(risk)}
+                />
+              ) : (
+                <li
                   style={
                     risk.isResolved
                       ? { textDecoration: 'line-through' }
                       : { textDecoration: 'none' }
                   }
+                  className="mb-3"
                 >
                   {risk.detail}
-                </p>
-              }
-              checked={risk.isResolved}
-              data-testId={`testCheckbox${idx}`}
-              onChange={() => handleCheck(risk)}
-            />
-            {risk.isResolved ? (
-              <OverlayTrigger overlay={renderTooltip('Delete Risk')}>
-                <Button
-                  variant="danger"
-                  data-testId="deleteButton"
-                  onClick={() => handleDelete(risk.id)}
-                >
-                  <FontAwesomeIcon icon={faTrash} />
-                </Button>
-              </OverlayTrigger>
-            ) : (
-              <OverlayTrigger overlay={renderTooltip('Convert to CR')}>
-                <Button
-                  variant="success"
-                  data-testId="convertButton"
-                  onClick={() => {
-                    history.push(
-                      routes.CHANGE_REQUESTS_NEW_WITH_WBS +
-                        wbsPipe(wbsNum) +
-                        '&riskDetails=' +
-                        encodeURIComponent(risk.detail)
-                    );
-                  }}
-                >
-                  <FontAwesomeIcon icon={faArrowRight} />
-                </Button>
-              </OverlayTrigger>
-            )}
-          </div>
-        ))}
-        <div>
-          <Button variant="success" onClick={handleShow}>
-            Add New Risk
-          </Button>
+                </li>
+              )}
+              {risk.isResolved ? DeleteRiskButton(risk) : ConvertToCRButton(risk)}
+            </div>
+          ))}
+          {hasPermissions ? (
```

Comment:
actually the best way to do this is just { hasPermissions && (<Button > ... </Button> }